NEWS

```diff
@@ -489,6 +489,11 @@ CHANGES WITH 257:
           existing interfaces, and invoke 'networkctl reload' or restart
           systemd-networkd.
 
+        * systemd-networkd now supports configuring the timeout for IPv4
```

Comment:
I noticed the entry is added in the section for v257.
Please mention the change in v258.